testsuite/unittests/test_symmetric_connections.sli

```diff
@@ -29,14 +29,16 @@ Synopsis: (test_symmetric_connections) run -> NEST exits if test fails
 
 Description: 
 This test ensures that the functionality to create symmetric connections
-works properly.
+works properly. It also ensures that the flag requires_symmetric works properly.
```

Comment:
This is confusing, since `requires_symmetric` is built-in property of synapse models, not a flag visible on the SLI level. On the other hand, we have the `/symmetric` flag very visible in this test.